velox/docs/monitoring/stats.rst

```diff
@@ -39,3 +39,25 @@ Memory Arbitration
      - kByte
      - The reclaimed memory bytes of an operator during the memory arbitration.
        This stats only applies for spillable operators.
+
+Task Execution
+------------------
+.. list-table::
+   :widths: 40 10 50
+   :header-rows: 1
+
+   * - Stats Name
+     - Type
+     - Description
+   * - hashtable.capacity
```

Comment:
The stats above are reported for any kind of operator. However, these stats are reported only by HashBuild and HashAggregation operators. 

Let's figure out how to clarify this in the docs. One option might be to create a section per operator.

Reply:
Add a `HashTable` section as they are used in both HashBuild and HashAggregation operators and `HashTable` is a very critical component.